distribution/rules.bzl

```diff
@@ -176,8 +177,8 @@ def distribution(targets,
     )
 
     tgz2zip(
-        name = "distribution",
-        tgz = ":distribution_tgz",
+        name = "{}".format(name),
```

Comment:
@lolski
```suggestion
        name = name,
```

Reply:
lol good spot why am i even doing that

Reply:
solved in https://github.com/graknlabs/deployment/pull/19/commits/554b1177084911f50151f7b8788c87c315cab67a